src/foundry/client/pixi/placeable.d.ts

```diff
@@ -35,7 +35,7 @@ declare global {
     /**
      * The underlying data object which provides the basis for this placeable object
      */
-    data: D['data'];
+    data: D['_source'];
```

Comment:
Looks like this is deprecated and `D` not `D['_source']`.

```js
    Object.defineProperty(this, "data", {
      get: () => {
        const msg = "You are accessing PlaceableObject#data which is no longer used and instead the Document class "
          + "should be referenced directly as PlaceableObject#document.";
        foundry.utils.logCompatibilityWarning(msg, {since: 10, until: 12});
        return document;
      }
    });
```

Just take the deprecation message from Foundry's message I think.